test/e2e/lib/flows/login-flow.js

```diff
@@ -204,7 +205,11 @@ export default class LoginFlow {
 			await sideBarComponent.searchForSite( siteURL );
 		}
 
-		return await StatsPage.Expect( this.driver );
+		if ( this.account.username !== 'e2eflowtestinggutenbergsimpleedge' ) {
+			await StatsPage.Expect( this.driver );
+		} else {
+			await CustomerHomePage.Expect( this.driver );
```

Comment:
Oh interesting, we likely have a userId check for customer home (available to new users). We can maybe drop the requirement for branching if we create other fresh test users.